openreview/conference/templates/tabsConferenceWebfield.js

```diff
@@ -72,13 +71,13 @@ function load() {
       details: 'forumContent,writable'
     });
 
-    userGroupsP = Webfield.getAll('/groups', { member: user.id, web: true })
-      .then(function(groups) {
-        return _.filter(
-          _.map(groups, function(g) { return g.id; }),
-          function(id) { return _.startsWith(id, CONFERENCE_ID); }
-        );
-      });
+    userGroupsP = Webfield.getAll('/groups', { regex: CONFERENCE_ID + '/.*', member: user.id, web: true })
```

Comment:
Is adding the regex param just a performance optimization? I see that we are filtering out groups that don't start with the conference ID below. Also, if a user is a member of CONFERENCE_ID, will this regex exclude the group CONFERENCE_ID since it has a '/' on the end?

Reply:
this userGroupP is being used 'only' to render the console so we should find all the group that exist after CONFERENCE_ID, I will remove the filtering because it doesn't make sense now.